### PR TITLE
Added tslint-config and prettier-config

### PR DESCRIPTION
**Pull request by 9renpoto**
- [x] travisを動かす
- [x] prettier-config も足す

異論はここで受け止めましょう

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (chantlong) [inline lerna.json]: lerna初耳ー

Comment (9renpoto) [reply lerna.json]: its using babel, and more

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (9renpoto) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: generated from github

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (9renpoto) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: generated from github

**Pull request merged**